src/py_sonar_scanner/configuration.py

```diff
@@ -60,6 +60,9 @@ def _read_wrapper_arguments(self):
         argument_parser.add_argument("-toml.path", "-Dtoml.path", "--toml.path", dest="toml_path")
         argument_parser.add_argument("-project.home", "-Dproject.home", "--project.home", dest="project_home")
         argument_parser.add_argument("-X", action="store_true", dest="debug")
+        argument_parser.add_argument(
+            "-read.toml", "-Dread.toml", "--read.toml", "-read_toml", action="store_true", dest="read_toml"
```

Comment:
Not sure what the name should actually be, this is placeholder.

Also unsure whether the default value should be true or false.

While `tool.sonar` properties will have precedence over this, I'm mostly afraid that properties deduced from seemingly random fields would have precedence over the content of a `sonar-project.properties` file (and I feel like parsing those to check for duplicate properties might be overengineered).

Reply:
I renamed to `read_project_config` which I guess may be a bit better, but I'm still not convinced.